cli/templates/tests/index.md

```diff
@@ -129,10 +129,12 @@ The file supports complex values and can even refer to an OAuth configuration:
 }
 ```
 
+## Sensitive Values
+
 Sensitive values can be stored in environmental variables and referred to using a placeholder encapsulated by `##`. 
 The variables need to be prefixed by `KBC_SECRET_`. 
```

Comment:
Tu by bolo dobre nejak vysvetlit, ze ide pouzit aj non-secret ENVs a ze tie nie su obmedzene.
Tj. mohla by byt nejaka sekcia napr. `Placeholders` a `Sensitive Values` by bol iba pod-nadpis.

+ hodil by sa tu 2x link na to, ako nastavit ENVs v Github Actions a ako secrets (pre Moniku).

Reply:
Třeba tak? https://github.com/keboola/developers-docs/pull/225/commits/e7b8b3a911d993939ba8fc0fcb61d5c6f5706d7b

Reply:
super :+1: